drakrun/drakrun/playground.py

```diff
@@ -119,11 +134,19 @@ def main():
     with graceful_exit(start_dnsmasq(args.vm_id, args.dns)), \
             DrakmonShell(args.vm_id, args.dns) as shell:
         helpers = {
+            'help': shell.help,
             'copy': shell.copy,
+            'mount': shell.mount,
             'drakvuf': shell.drakvuf,
             'vm': shell.vm
         }
-        embed(banner='', user_ns=helpers, colors='neutral')
+        banner = """
+        *** Welcome to drakrun playground ***
+        Your vm is now ready and running with internet connection.
+        You can connect to it using vnc (password can be found in /etc/drakrun/configs/vm-0.cfg)
```

Comment:
```suggestion
        You can connect to it using VNC (password can be found in /etc/drakrun/scripts/cfg.template)
```

Reply:
@chivay won't `drak-vncpasswd` be better here?

Reply:
technically yes, but it may not be installed - it's in the other package.
I think that we may fix this later, i.e:
* move drak-vncpasswd to drakrun
* call it directly (or some utility function) directly from drakplayground to display VNC password